pkg/api/validation/schema.go

```diff
@@ -62,6 +64,70 @@ type NullSchema struct{}
 
 func (NullSchema) ValidateBytes(data []byte) error { return nil }
 
+type NoDoubleKeySchema struct{}
+
+func (NoDoubleKeySchema) ValidateBytes(data []byte) error {
```

Comment:
Unfortunate that you have to hardcode just two fields.  This would be a lot more useful if it traversed all maps.